packages/amplify_authenticator/lib/src/l10n/generated/input_localizations_en.dart

```diff
@@ -29,6 +29,9 @@ class AuthenticatorInputLocalizationsEn
   @override
   String get password => 'Password';
 
+  @override
+  String get newPassword => 'New Password';
```

Comment:
Previously we were re-using the content for password, so the hint text was "Enter your password", which is confusing because the form was forcing a new password.